server/export.js

```diff
@@ -27,9 +27,12 @@ module.exports = function(fs, db) {
             });
         }) // run the command to output the table to a file
         // replace 'annotation' with a select query to get certain rows instead of the whole thing
-        .then(db.none(`copy annotation to '/tmp/csv/export.csv' DELIMITER ',' CSV HEADER`))
+        .then(db.none(`COPY annotation TO '/tmp/csv/export.csv' DELIMITER ',' CSV HEADER`))
         .then(function() {
             // download file to client
+            if (!fs.existsSync('/tmp/csv/export.csv')){
+                fs.writeFile('/tmp/csv/export.csv', 'content')
```

Comment:
any error thrown should be caught by the catch() statement and prevent the server from crashing, unless you tried it at some point and it actually did crash. I'm wary of stuff like this because we've all had that situation where we make something that's just for a prototype and it ends up staying in the code for way longer than it should, I would rather get rid of this now than risk forgetting about it later